lib/Runtime/SerializableFunctionFields.h

```diff
@@ -71,6 +71,7 @@ PRIVATE_FIELDS
     DECLARE_SERIALIZABLE_FIELD(bool, m_hasFinally, Bool);
     DECLARE_SERIALIZABLE_FIELD(bool, hasScopeObject, Bool);
     DECLARE_SERIALIZABLE_FIELD(bool, hasCachedScopePropIds, Bool);
+    DECLARE_SERIALIZABLE_ACCESSOR_FIELD(uint, ForInLoopDepth, UInt32);
```

Comment:
>UInt32 [](start = 62, length = 6)

ForInLoopDepth will be smaller than or equal to ProfiledForInLoopCount. Why are we giving a higher type e to ForInLoopDepth?